### DISCUSSION TITLE
Logo and Release Date

### DISCUSSION THREAD

**Discussion by sayanarijit** [ratatui/ratatui#86] [Polls]
We are not too far from the initial release. One last major thing to do (for promotion) before the release is adding a logo.

It doesn't have to be the "BEST LOGO EVER". Anything contributed by the community, that's not subject to copyright issues, should do fine. So, please submit your creations in this thread, and vote on your preference using the emojis. We already have quite a few cool submissions from the discord community. You can take them as inspiration for your logo.

We will pick the most popular logo just before the initial release, which will also be decided in this poll.

**Comment by sayanarijit** [comment]
by @UncleScientist

![image](https://user-images.githubusercontent.com/11632726/224469005-3c3aadf0-8688-4487-b0b3-5b385402da86.png)

**Comment by sayanarijit** [comment]
by @orhun

![image](https://user-images.githubusercontent.com/11632726/224469039-4c40f51b-644f-4031-8001-543313b096fb.png)

**Comment by sayanarijit** [comment]
by @orhun 

![image](https://user-images.githubusercontent.com/11632726/224469051-56ec1c96-35ae-4b1d-9ee6-307ec12141fb.png)

**Reply by nidnogg** [reply]
for me, it's this guy 
EDIT: Replaced with @nawok's ASCII + shell suggestion

**Reply by mindoodoo** [reply]
I feel like this one has the nice minimalistic logo vibe

**Comment by sayanarijit** [comment]
by @diamant3

![image](https://user-images.githubusercontent.com/11632726/224469059-9daa6a84-7445-4f44-b7c2-4aa5be15bc20.png)

**Comment by sayanarijit** [comment]
by @UncleScientist's son

![image](https://user-images.githubusercontent.com/11632726/224469397-9547631d-7209-4752-8d3e-ff8dbb0a8f6a.png)

**Comment by sayanarijit** [comment]
by @UncleScientist's son

![image](https://user-images.githubusercontent.com/11632726/224469432-4611cb20-b6cb-4116-a10d-0415fb4fe357.png)

**Reply by mindoodoo** [reply]
I like this one, although I'm sure we can find someone to polish up the rough edges at some point (I literally only mean the pixelated edges haha)

**Comment by sayanarijit** [comment]
by @UncleScientist's son

![image](https://user-images.githubusercontent.com/11632726/224469470-5d6c8fa1-f9be-4695-a4e9-044933f77e1d.png)

**Comment by sayanarijit** [comment]
by @UncleScientist's son

![image](https://user-images.githubusercontent.com/11632726/224469508-bbd9574a-ba4d-4636-8488-6d531ec901b7.png)

**Comment by sayanarijit** [comment]
A sad Ratatui by @UncleScientist

![image](https://user-images.githubusercontent.com/11632726/224469534-d12bc82d-7b50-4879-adeb-a79503a94553.png)

**Comment by nawok** [comment]
Can’t think about anything original and this mouse just lives in my head. I tried to mimic TUI’s graphics with dot matrix. Unfortunately this cannot be downscaled “as-is” and additional variants need to be done for smaller sizes (e.g. Discord logo or GitHub org) including the light theme.

Just wanted to share the idea. I borrowed the colors from the terminal from the logo of @UncleScientist’s son. :)

See below:

**Reply by sayanarijit** [reply]
Nice... Thanks for your contribution. I have moved the images from here to separate comments so that it's easier to vote on.

**Comment by nawok** [comment]
#### Replaced the spoon with a terminal

![ratatui-terminal-256](https://user-images.githubusercontent.com/159773/224647258-8b7fd44e-0c6d-4ebd-9898-ecbeeef9c693.png)

**Reply by nidnogg** [reply]
Replacing my previous vote with this

**Reply by VelvetToroyashi** [reply]
I personally think the terminal would look better straight, but it's a nice logo regardless

**Reply by sayanarijit** [reply]
> I personally think the terminal would look better straight, but it's a nice logo regardless

I like it aligned, though...

**Reply by orhun** [reply]
really good logo.

**Comment by sayanarijit** [comment]
Colored by @nawok 

![image](https://user-images.githubusercontent.com/11632726/224599205-5ee0de24-125c-4d5f-8d05-3a9a305a0fe5.png)

**Comment by sayanarijit** [comment]
1-bit by @nawok 

![image](https://user-images.githubusercontent.com/11632726/224599270-ca39049a-3374-4bed-9de5-5b0d3ac80241.png)
